### PR TITLE
Added text file

### PR DESCRIPTION
**Pull request by otsembo**
Extra information .....
```ruby
puts "hello world"
```
icons8-great-64

### CODE REVIEW

**Review by otsembo — COMMENTED** [review]
Good Job @otsembo 
I need you to fix:

- Feature A
- Feature B

File: ruby_test.txt [hunk 1]
```diff
@@ -0,0 +1 @@
+Hello from branch ruby
```
Comment (otsembo) [inline ruby_test.txt:1]: Change this text.

**Review by otsembo — COMMENTED** [review]
Added one more fix

**Review by otsembo — COMMENTED** [review]
Great job. Everything looks good.
LGTM (Looks good to me)

File: ruby_test.txt [hunk 2]
```diff
@@ -0,0 +1,2 @@
...
+Extra words
```
Comment (Rhonajoy) [inline ruby_test.txt:2]: add a proper heading.

Comment (monginadiana) [inline ruby_test.txt:2]: will fix it soon

**Review by Rhonajoy — APPROVED** [review]
LGTM

**Review by monginadiana — COMMENTED** [review]
will make the fix soon enough

File: ruby_test.txt (hunk 2 above)
Comment (monginadiana) [reply ruby_test.txt:2]: will add the heading soon enough

**Review by monginadiana — CHANGES_REQUESTED** [review]
noted on the changes

File: ruby_test.txt (hunk 1 above)
Comment (otsembo) [reply ruby_test.txt:1]: Fixed

**Pull request merged**